### PR TITLE
Add function emitter event to Radiswap

### PR DESCRIPTION
**Pull request by 0xOmarA**
## Summary

A small PR that adds a single event to Radiswap that uses the function emitter as requested by the Gateway team.

### CODE REVIEW

**Review by talekhinezh — APPROVED** [review]
Looks good, just some broken tests for some reason

**Comment by 0xOmarA** [comment]
Thanks @talekhinezh, the tests that are failing is due to the recent changes to the windows CI runners that devops are making. Waiting for them to fix this issue before I merge.

**Pull request merged**
